doc/getting-started/getting-started.rst

```diff
@@ -533,7 +536,7 @@ Build ACRN
 
       cd ./build
       ls *.deb
-         acrn-my_board-shared-2.7.deb # TODO update file name
+         acrn-my_board-MyConfiguration-3.0-acrn-*.deb # TODO update file name
```

Comment:
If that's indeed the right file name now, remove the #TODO comment

Reply:
update